CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Fixed
+
+- Brightness for 72x40 display.
+
+### Added
+
+- SSD1306B datasheet.
+
```

Comment:
Please add a link to the PR here. I also don't think we need to show added docs in the changelog.

```suggestion
### Fixed

- [#141](https://github.com/jamwaffles/ssd1306/pull/141) Brightness for 72x40 display.
```